src/Makefile.am

```diff
@@ -675,11 +675,17 @@ repl_modules.cc: repl_modules.sh Makefile
 
 include $(top_srcdir)/doc/manuals/Substitute.am
 
-squid.8: $(srcdir)/squid.8.in Makefile
-	$(SUBSTITUTE) < $(srcdir)/squid.8.in > $@
+squid.md: $(srcdir)/squid.md.in Makefile
+	$(SUBSTITUTE) < $(srcdir)/squid.md.in > $@
+
+doc_DATA = squid.md
+EXTRA_DIST += squid.md.in
+CLEANFILES += squid.md
+
+squid.8: squid.md
+	$(PANDOC) -t man -s -V "title:squid" -V "section:8" -o $@ $<
```

Comment:
If possible, please extract `-V title:...` information automatically so that we do not have to add a custom PANDOC rule for every .8 target.

Reply:
Agreed, that would be ideal. For now I have not been able to make a `.8:` target that works reliably.

Reply:
> I have not been able to make a `.8:` target that works reliably.

What primary difficulty in computing "title:foo" do you need to overcome?

Reply:
That part is fine using `basename $@`.  The issue I am struggling against is automake not detection that it needs to be re-generated when the input `squid.mdfile is changed.

Reply:
Solved in [f46c4b2](https://github.com/squid-cache/squid/pull/1586/commits/f46c4b213b62e38cc8759f05c064dab14e871bee)